src/main/java/sirius/biz/storage/layer3/VirtualFileSystemController.java

```diff
@@ -222,6 +223,36 @@ public void delete(WebContext webContext) {
         }
     }
 
+    /**
+     * Deletes the given list of files or directories.
+     *
+     * @param webContext the request to handle
+     */
+    @LoginRequired
+    @Routed("/fs/delete/multiple")
+    @Permission(PERMISSION_VIEW_FILES)
+    public void deleteMultiple(WebContext webContext) {
+        List<String> filePaths = webContext.getParameters("paths");
+        filePaths.removeAll(Arrays.asList("", null));
+        AtomicReference<String> workingDirectory = new AtomicReference<>();
```

Comment:
In other places, where it must not be atomic, we use a `ValueHolder` for reduced overhead